.travis.yml

```diff
@@ -12,6 +12,7 @@ matrix:
     env: JEKYLL_VERSION=3.4.3
 env:
   matrix:
+  - JEKYLL_VERSION=3.3.0
```

Comment:
That will cause 3.4.3 to be installed (as the Gemfile specifies `~> #{ENV["JEKYLL_VERSION"]}`, so we'd be testing the same version in both environments. We want to test before `< 3.4.2`.